### ISSUE TITLE
Ability to see completed job history

### ISSUE THREAD

**Issue by DKS-18** [bambulab/BambuStudio#456]
Currently, there is no way of checking what prints have been completed on the printer.

when doing print jobs that involve multiple beds it would be good to be able to confirm what jobs have / have not been printed

**Comment by YifanWuBambu** [comment]
@DKS-18 
Yes. This is in our feature plan. Currently it is not supported yet.

**Comment by redlinebikeparts** [comment]
110% agree would be great to have this.

**Comment by joshmitcho** [comment]
@bambu123 any updates on this?

**Comment by sharri46** [comment]
bump

**Comment by ananggadipa** [comment]
While you guys are at it, please make the log downloadable as a CSV file with time, duration, filament, etc.

**Comment by itsonlymedan** [comment]
Bump

**Comment by lackovic** [comment]
The best way to raise attention on a GitHub issue is to thumb up the top post and subscribe to notifications.
